rocky/reports/report_types/definitions.py

```diff
@@ -107,6 +107,34 @@ def to_hostnames(self, input_oois: Iterable[str], valid_time: datetime) -> dict[
 
         return hostnames_by_input_ooi
 
+    def to_human_readable_hostnames(
+        self,
+        input_oois: Iterable[str],
+        valid_time: datetime,
+        extra_indicator: str = ", ...",
+        open_indicator: str = "(",
+        close_indicator: str = ")",
+        hostnames_by_input_ooi: dict | None = None,
+    ) -> dict:
+        """Converts input_oois to human readable hostname strings.
+
+        Turns a list of either Hostname and IPAddress references into a string
+        of shortest related hostname and indication on more hostnames present,
+        grouped by input ooi.
+        """
+        if hostnames_by_input_ooi is None:
+            hostnames_by_input_ooi = self.to_hostnames(input_oois=input_oois, valid_time=valid_time)
```

Comment:
`to_hostnames` method accepts positional arguments, so there's no need to pass them as keyword arguments

```suggestion
            hostnames_by_input_ooi = self.to_hostnames(input_oois, valid_time)
```

Reply:
Yeah, I followed the "explicit is better than implicit" doctrine here. There may be a [special syntax](https://peps.python.org/pep-0736/) someday available for this. However, if you prefer the positional arguments I can remove the keywords

Reply:
I would still prefer the positional arguments at least for consistency. I would agree to use keyword arguments for more complex signatures, but this one is relatively simple and by using them as positional arguments, it would minimise verbosity and visual noise.